### DISCUSSION TITLE
Why does Heroku not use Workers?

### DISCUSSION THREAD

**Discussion by hahnbeelee** [taskforcesh/bullmq#1559] [Q&A] [answered]
I have bull set up in my express app hosted on heroku. [Heroku's docs' example](https://devcenter.heroku.com/articles/node-redis-workers) uses queues. I have a job that takes a really long time and I saw BullMQ's docs mentioned using Workers for Sandboxed processors which I think might be helpful for my use case. Why doesn't Heroku mention using Workers? Should I not do this?

**Comment by hahnbeelee** [comment, marked as answer]
nvm i just came to the conclusion that bull != bullmq 😅

**Reply by manast** [reply]
Even though they are not the same library they work in a similar way, BullMQ being the newest codebase. The concept of Worker is not related to a sandboxed processor, a Worker is the actual "thing" that processes jobs. In BullMQ we have the class Worker for this, but in Bull there is only one class Queue, that does both.
